src/main/PDF/Services/CrudServices/UploadPDFService.java

```diff
@@ -107,7 +110,19 @@ public Message mongodbUpload() throws GeneralSecurityException, IOException {
                       .append("annotated", false)
                       .append("uploader", uploader)
                       .append("organizationName", organizationName));
-
+// sorry I made a whole new else if I couldn't figure out how to edit the metadata after the fact
+    } else if (pdfType == PDFType.IDENTIFICATION_DOCUMENT){
```

Comment:
why?

Reply:
I didn't write this - anyway, I refactored it to be consolidated and only add the metadata specific to the document type

Reply:
Or if I did write this, it was years ago